src/components/TopNav.js

```diff
@@ -1,15 +1,19 @@
 import React from "react";
 
-function TopNav() {
+function TopNav(props) {
+  const handleClick = event => {
+    event.preventDefault();
+    props.changeCategory(event.target.value);
+  }
   return (
     <div className="top-nav">
       <ul>
-        <li><a href="carlights.html">car lights</a></li>
-        <li><a href="carwheels.html">Car wheels</a></li>
-        <li><a href="carbumpers.html">car bumpers</a></li>
-        <li><a href="caradsystem.html">car audiosystem</a></li>
-        <li><a href="truckbumpers.html">Truck bumpers</a></li>
-        <li><a href="contact.html">Feedback</a></li>
+        <li><a href="#" onClick={handleClick}>car lights</a></li>
```

Comment:
Double check this approach. A tags do not have a value
All we need to do is the hard code the category name into the function call such as "tires" or "bumpers"